### ISSUE TITLE
No accessibility for screen reader software in editor

### ISSUE THREAD

**Issue by volkda** [nextcloud/richdocuments#2000]
I'M blind and I use a screen reader for working on computer. The text field editor of the collabora app is not accessible for the screen reader. I can write text but I cannot read the writtin text if I pressed the return key or I open the document later. It is not readable for the screen reader software. Please fix this and you will improve the accessibility.

**Comment by mmeeks** [comment]
Thanks for your feedback! we are working to improve accessibility of Collabora Online as a longer term feature development with several intersecting pieces here - such as migrating all dialogs to be client-side rendered, and improving web-aria annotation left & right. In the latest release (21.11.2) we've released our Accessibility Checker which allows the majority of users to check that they are creating accessible documents & PDFs to assist impaired users - making it easier to generate good documents that meet PDF/UA rules. We continue to look at how best to implement html/canvas accessibility in the best way for the editing core.

**Comment by juliushaertl** [comment]
Thanks for reporting this, as Michael mentioned there is tons of work that has happened since this report and still lots of improvements happening. I'll close this here as it is mostly an upstream topic of https://github.com/CollaboraOnline/online
